tests/oisp-tests.js

```diff
@@ -1136,7 +1135,27 @@ describe("Sending observations and checking rules ...\n".bold, function() {
        }).timeout(BACKEND_TIMEOUT);
 
 });
+describe("Do basic rule and alerts subtests ...".bold, function() {
 
+    var test;
```

Comment:
This looks likes it could take long, should be add a skip flag? @wagmarcel

Reply:
thanks @arkocal  - yes forgot to add the skip. Now added.